pysumma/pysumma/Plotting.py

```diff
@@ -0,0 +1,111 @@
+import seaborn as sns
+import xarray as xr
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+class Plotting:
+
+	colors = ["windows blue", "amber", "greyish", "faded green", "dusty purple"]
+	sns.set_palette(sns.xkcd_palette(colors))
+
+	def __init__(self, filepath):
+		self.filepath = filepath
+
+	def open_netcdf(self):
+		filepath = self.filepath
+		ds = xr.open_dataset(filepath)
+		print(ds)
+		return ds
+
+	def plot_1d(self, ds, varibale_num):
```

Comment:
`varibale` -> `variable`
Also, why use the variable number as the key instead of the name?